components/director/internal/open_resource_discovery/service_test.go

```diff
@@ -1561,6 +1641,10 @@ func TestService_SyncORDDocuments(t *testing.T) {
 	for _, test := range testCases {
 		t.Run(test.Name, func(t *testing.T) {
 			_, tx := test.TransactionerFn()
+			labelRepo := &automock.LabelRepository{}
```

Comment:
Add it to AssertExpectations below or it won't do anything.

Reply:
Good catch 👍